### PR TITLE
Add label class

### PR DESCRIPTION
**Pull request by c00p75**
- Create a ```class Label``` with an association to the Item class.
- All Label class properties visible in the diagram should be defined and set up in the constructor method.
- Implement add_item method in the ```class Label```

### CODE REVIEW

File: label.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+  def initialize(title, color)
+    @id = Random.rand(1..1000)
+    @title = title
+    @color = color
+    @items = []
+  end
+
+  def add_item(item)
+    @item << item unless @items.include?(item)
```
Comment (Tafloninno) [inline label.rb]: Used `@item` instead of `@items`. This appears to be a typo as it should be `@items`. kindly fix that ``

**Review by Tafloninno — CHANGES_REQUESTED** [review]
Hi @c00p75 ,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Highlights
✅ Linters passing
✅ Good commit messages

Required Changes ♻️
Check the comments under the review.

Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

File: label.rb (hunk 1 above)
Comment (c00p75) [reply label.rb]: Great observation. Thank you for this. I'll be sure to correct it.

**Review by Tafloninno — APPROVED** [review]
Status: APPROVED 💯
Hi @c00p75  ,

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

Highlights
✔️ Linters passing
✔️ Good commit messages

Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**